src_nbgl/ui_signing.h

```diff
@@ -0,0 +1,21 @@
+#ifndef UI_SIGNING_H_
```

Comment:
pragma once ?

Reply:
The app codebase uses old-school include guards, so I stuck with it for consistency. :+1: